helm/dagster/schema/schema/charts/dagster/subschema/daemon.py

```diff
@@ -68,6 +68,8 @@ class Daemon(BaseModel):
     env: Dict[str, str]
     envConfigMaps: List[kubernetes.ConfigMapEnvSource]
     envSecrets: List[kubernetes.SecretEnvSource]
+    labels: Dict[str, str]
+    podLabels: Dict[str, str]
```

Comment:
so we already have `labels` in the run launcher and they apply to the pod not the job - given that i think we should swap which one is the default? (so labels and deploymentLabels)?